core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -133,15 +131,27 @@ func TestLogPoller_Integration(t *testing.T) {
 		require.NoError(t, err1)
 		th.Client.Commit()
 	}
+	// Calling Start() after RegisterFilter() simulates a node restart after job creation, should reload Filter from db.
+	require.NoError(t, th.LogPoller.Start(testutils.Context(t)))
+
 	// The poller starts on a new chain at latest-finality (5 in this case),
```

Comment:
The 5 in this comment was incorrect before, due to `Start()` being called after block 2 is committed.  Should be accurate now that `Start()` has been moved to after blocks 3, 4, 5, 6, and 7 are been committed.